packaging/nmstate.spec

```diff
@@ -35,6 +35,7 @@ Recommends:     NetworkManager-config-server
 Suggests:       NetworkManager-ovs
 Suggests:       NetworkManager-team
 Requires:       python3-nispor
+Requires:       python3dist(varlink)
```

Comment:
This should not be necessary since requirements.txt are considered by rpmbuild.

Reply:
Removed, thanks.